ilp-core/src/test/java/org/interledger/core/InterledgerResponsePacketTest.java

```diff
@@ -0,0 +1,104 @@
+package org.interledger.core;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.fail;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * Unit tests for {@link InterledgerResponsePacket}.
+ */
+public class InterledgerResponsePacketTest {
+
+  private static final byte[] EMPTY_DATA = new byte[64];
```

Comment:
Curious: Any specific reason why this is a `64` byte array instead of the usual `32`?

Reply:
No specific reason. Could be data/size, really. Probably a better name would be just `DATA` since "empty data" usually looks like `new byte[]`.